packages/babel-traverse/src/scope/index.js

```diff
@@ -140,7 +140,7 @@ const collectorVisitor = {
   },
 
   UpdateExpression(path, state) {
-    state.constantViolations.push(path.get("argument"));
+    state.constantViolations.push(path);
   },
 
   UnaryExpression(path, state) {
```

Comment:
Should this also apply to the `delete` operator? It appears the actual property is being placed as the violation rather than the operation itself.